app/src/lib/WalletProvider.svelte

```diff
@@ -1,18 +1,22 @@
-<script context="module" lang="ts">
+<!-- <script context="module" lang="ts">
 	import { Buffer } from 'buffer';
 	import process from 'process';
 
 	globalThis.Buffer = Buffer;
 	globalThis.process = process;
-</script>
 
+	if (typeof window !== 'undefined') {
+		window.global = window;
+	}
+</script> -->
 <script lang="ts">
-	import type { Wallet } from '@solana/wallet-adapter-wallets';
+	// import type { Wallet } from '$utils/walletStore';
+	import type { Adapter } from '@solana/wallet-adapter-base';
 	import { initialize } from '$utils/walletStore';
 	import type { WalletError } from '@solana/wallet-adapter-base';
 
 	export let localStorageKey: string,
-		wallets: Wallet[],
+		wallets: Adapter[],
```

Comment:
This is the biggest change here. Following what is done here https://github.com/solana-labs/wallet-adapter/pull/267/files it seems like that adapter and wallet are the same. The Vue code uses adapter for internal operations, and expose to outside `wallet`, but essentially is the same. It is up to us to rename to adapter everything regarding wallet.